### PR TITLE
GH-37: Tool - list imported libraries

### PR DESCRIPTION
**Pull request by unkmc**

### CODE REVIEW

File: ghidra-assistant/.project
Comment (cyberkaida) [inline ghidra-assistant/.project:1]: Please split this and the class path changes into another PR.

File: ghidra-assistant/src/main/java/reva/RevaService.java [hunk 1]
```diff
@@ -103,7 +101,7 @@ public void sendToReva(RevaMessage message) {
      * @throws RevaServerException if there is a problem communicating with ReVa
      */
     private RevaMessage getMessageFromServer() throws RevaServerException {
-        URI endpoint = revaServerBase.resolve("/project/" + currentProgram.getName() + "/message");
+        URI endpoint = revaServerBase.resolve("/project/" + currentProgram.getDomainFile().getName() + "/message");
```
Comment (cyberkaida) [inline ghidra-assistant/src/main/java/reva/RevaService.java:104]: Reminder to update the README.md for this change.

**Review by cyberkaida — COMMENTED** [review]
Just one quick review while this is in draft. Thank you for the contribution, it's looking very good!

I would prefer to split the eclipse part into another PR so I can test the new message types (I don't use eclipse for Ghidra development).

File: ghidra-assistant/src/main/java/reva/RevaService.java (hunk 1 above)
Comment (unkmc) [reply ghidra-assistant/src/main/java/reva/RevaService.java:104]: I should put this context here too: 
I ran into a problem where I had two files in a project, `swtor.exe` and `swtor.exe_2024xxyy`. I was working in `swtor.exe_2024xxyy` and `currentProgram.getName()` was returning `swtor.exe`, so the Ghidra plugin kept hitting the wrong `reverse-engineering-assistant` API endpoint. After exploring the Ghidra API for a while, I eventually found that `currentProgram.getDomainFile().getName()` would return the correct `swtor.exe_2024xxyy` name.

**Comment by unkmc** [comment]
too out of date now

**Pull request closed without merging**